transformations/contraction_expansions/transformation.py

```diff
@@ -0,0 +1,227 @@
+import re
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+"""
+Contractions and Expansions List(s) borrowed from https://github.com/marcotcr/checklist/blob/master/checklist/perturb.py
+"""
+
+
+class ContractionExpansions(SentenceOperation):
+    tasks = [TaskType.TEXT_CLASSIFICATION, TaskType.TEXT_TO_TEXT_GENERATION]
+    languages = ["en"]
+
+    def __init__(self):
+        super().__init__()
+
+    def contract(self, sentence):
+        """Contract expanded contractions in a sentence (if any)
+        Parameters
+        ----------
+        sentence : str
+            input string
+        Returns
+        -------
+        string
+            String with contractions contracted (if any)
+        """
+        reverse_contraction_map = {
```

Comment:
I would suggest moving this map to the `__init__()` function so that it wouldn't get loaded everytime the `generate()` function is called.

Reply:
Another question: Is this `reverse_contraction_map` the same as the `contraction_map` but just reversing the key-value?

If yes, I think you can simply do a dict comprehension rather than hard-coding it twice.
```
contraction_map = {.....}
reverse_contraction_map = {value:key for key, value in contraction_map.items()}
```